cos-gpu-installer-docker/gpu_installer_url_lib.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
Could you run this file through shellcheck? I'm curious what it will say

Reply:
Done. Code is changed according to shellcheck advice.